modules/custom/optional/govcms8_default_content/govcms8_default_content.info.yml

```diff
@@ -0,0 +1,13 @@
+name: GovCMS 8 Default Content
```

Comment:
I'll organise the service design/comms team to go over the text used for the default content